platform/error/platform-error-page-injector/src/main/java/org/codice/ddf/platform/error/injector/ErrorPageInjector.java

```diff
@@ -82,6 +82,7 @@ public void event(ServiceEvent event, Map<BundleContext, Collection<ListenerInfo
 
   public void init() {
     checkForMissedServletContexts();
+    LOGGER.trace("skipping error page initialization.");
```

Comment:
The class reflection the error page injector used no longer works under j21. But registering a global error page is now easier with the new pax web version. So that should be done instead of this method.